saleor/product/migrations/0102_merge_20190802_1110.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.3 on 2019-08-02 16:10
```

Comment:
Do we need that? It is not released yet so I assume that you can remove your migration and create the new one.

Reply:
While rebaseing I've removed the merge migration and but my on top of the latest one.